### ISSUE TITLE
TTML color coding

### ISSUE THREAD

**Issue by btsimonh** [IMFTool/IMFTool#13]
Please check the HTML rendering for color from ttml.

The following HTML (from imftool):
```
<p begin="00:01:25.360" end="00:01:27.360" region="region_1" style="color:#FFFFFFFF;font-family:proportionalSansSerif;font-size:1.5c;font-style:normal;line-height:125%;text-align:start;" xml:id="8">
<span style="color:#FF0000FF;font-family:proportionalSansSerif;font-size:1.5c;font-style:normal;line-height:125%;text-align:start;">Red</span>
<span style="color:#00FF00FF;font-family:proportionalSansSerif;font-size:1.5c;font-style:normal;line-height:125%;text-align:start;"> Green</span>
<span style="color:#FFFF00FF;font-family:proportionalSansSerif;font-size:1.5c;font-style:normal;line-height:125%;text-align:start;"> Yellow</span>
<span style="color:#FF00FFFF;font-family:proportionalSansSerif;font-size:1.5c;font-style:normal;line-height:125%;text-align:start;"> Magenta</span>
<span style="color:#00FFFFFF;font-family:proportionalSansSerif;font-size:1.5c;font-style:normal;line-height:125%;text-align:start;"> Cyan</span>
<span style="color:#0000FFFF;font-family:proportionalSansSerif;font-size:1.5c;font-style:normal;line-height:125%;text-align:start;"> Blue</span>
<br style="color:#0000FFFF;font-family:proportionalSansSerif;font-size:1.5c;font-style:normal;line-height:125%;text-align:start;"/>
<span style="color:#FFFFFFFF;font-family:proportionalSansSerif;font-size:1.5c;font-style:normal;line-height:125%;text-align:start;">WhiteWithCode</span>
<br style="color:#FFFFFFFF;font-family:proportionalSansSerif;font-size:1.5c;font-style:normal;line-height:125%;text-align:start;"/>
<span style="color:#FFFFFFFF;font-family:proportionalSansSerif;font-size:1.5c;font-style:normal;line-height:125%;text-align:start;">RedBackground</span>
<span style="color:#FFFFFFFF;font-family:proportionalSansSerif;font-size:1.5c;font-style:normal;line-height:125%;text-align:start;"> BlackBgndShortcode</span>
</p>
```
results in:

![image](https://user-images.githubusercontent.com/3137332/56204919-69295280-6040-11e9-924c-4a05ab5048dd.png)

but should be like:
![image](https://user-images.githubusercontent.com/3137332/56205098-c2918180-6040-11e9-9bd5-43c5ce1e734e.png)

I believe the #RRGGBBAA is being interpreted as #AARRGGBB
https://drafts.csswg.org/css-color/#hex-notation

this is almost certainly a difference between QT and current browsers:
https://doc.qt.io/qt-5/qml-color.html

The ttml file:
[testfilepresentation-DNK.zip](https://github.com/IMFTool/IMFTool/files/3084692/testfilepresentation-DNK.zip)

p.s. it would be nice if the TTML box rendered over a grey background so that the background style of the text can be seen.

**Comment by IMFTool** [comment]
Addressed by [V1.5.3](https://github.com/IMFTool/IMFTool/releases/tag/1.5.296)

**Comment by btsimonh** [comment]
:)
